content/cloud/variables.md

```diff
@@ -74,18 +74,21 @@ HTTP path: /...
 
 ## Adding Variables to an Application Component
 
-Our application receives a password via the HTTP request body, compares it to an expected password, and returns a JSON response indicating whether the submitted password matched or not.
+Our application receives a password via the HTTP request body, compares it to an expected password, and returns a JSON response indicating whether the submitted password matches or not.
 
-In reality, you'd have multiple usernames and password hashes in a database, but for this tutorial we will configure the expected password using a Spin application variable. We'll name the variable `password` and set `required = true` since there is no reasonable default value. To do this, add a top-level `[variables]` section to the application manifest (`spin.toml`) and declare the variable within it.
+In reality, you'd have multiple usernames and password hashes in a database, but for this tutorial, we will configure the expected password using a Spin application variable. We'll name the variable `password` and set `required = true` since there is no reasonable default value. To do this, add a top-level `[variables]` section to the application manifest (`spin.toml`) and declare the variable within it:
+
+<!-- @nocpy -->
```

Comment:
Not sure why these sections are being marked no-copy?

Reply:
Thanks @itowlson 
Originally just thought of limiting coping functionality to commands only. I have updated now so that relevant config and code can be copied by the reader/user. Thanks for spotting and mentioning it.